### DISCUSSION TITLE
Files hidden by readme.

### DISCUSSION THREAD

**Discussion by Y-Less** [community/community#41969] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I posted this a few days ago when it first started happening, but the topic seems to have vanished and the problem still occurs.  In fact it has got worse.

Previously opening a repo would 1/10 times cover up the list of files with the repo and this video took a while to capture:

![vanish7](https://user-images.githubusercontent.com/991078/208236846-f8d9ed92-598f-4bfa-b614-5306d8ea2413.gif)

Now it happens every time and I can't get to any files.

**Comment by jfullstackdev** [comment]
can you post the link of that ? because I will test it on my end

just by looking at your commits, that's a very big file and if you have low memory and slow internet connection that will take time to load

**Comment by Y-Less** [comment]
Well that was only an example, it happens on every repo.  This is the view I'm looking at right now:

![image](https://user-images.githubusercontent.com/991078/208256242-315e12ed-cfb5-48f2-9c01-ba9548e70427.png)

On this repo: https://github.com/Y-Less/samp-plugin-sdk which is very small and doesn't even have a readme, so the file list appears for a second then vanishes (which also means I doubt it is a speed issue because the data is loaded first then goes).  And it worked fine for years until about a week ago.

**Reply by jfullstackdev** [reply]
try Incognito, 

how about other browsers ?

**Reply by Y-Less** [reply]
Chrome seems to work, private browsing doesn't.
